Sonic-06-Mod-Manager/res/Applications/data/keys

```diff
@@ -1,10 +1,2700 @@
+self_type=LV2
+erk=0CAF212B6FA53C0DA7E2C575ADF61DBE68F34A33433B1B891ABF5C4251406A03
+riv=9B79374722AD888EB6A35A2DF25A8B3E
+pub=1034A6F98AF6625CC3E3604B59B971CA617DF337538D2179EBB22F3BDC9D0C6DA56BA7DDFD205A50
+priv=009D4CBA2BFB1A8330D3E20E59D281D476D231C73A
+ctype=14
+
+[lv2ldr]
+type=SELF
+revision=0000
+version=0004007600000000
+self_type=LV2
+erk=0CAF212B6FA53C0DA7E2C575ADF61DBE68F34A33433B1B891ABF5C4251406A03
+riv=9B79374722AD888EB6A35A2DF25A8B3E
+pub=1034A6F98AF6625CC3E3604B59B971CA617DF337538D2179EBB22F3BDC9D0C6DA56BA7DDFD205A50
+priv=009D4CBA2BFB1A8330D3E20E59D281D476D231C73A
+ctype=14
+
+[lv2ldr]
+type=SELF
+revision=0000
+version=0004007800000000
+self_type=LV2
+erk=0CAF212B6FA53C0DA7E2C575ADF61DBE68F34A33433B1B891ABF5C4251406A03
+riv=9B79374722AD888EB6A35A2DF25A8B3E
+pub=1034A6F98AF6625CC3E3604B59B971CA617DF337538D2179EBB22F3BDC9D0C6DA56BA7DDFD205A50
+priv=009D4CBA2BFB1A8330D3E20E59D281D476D231C73A
+ctype=14
+
+[lv2ldr]
+type=SELF
+revision=0000
+version=0004008000000000
+self_type=LV2
+erk=0CAF212B6FA53C0DA7E2C575ADF61DBE68F34A33433B1B891ABF5C4251406A03
+riv=9B79374722AD888EB6A35A2DF25A8B3E
+pub=1034A6F98AF6625CC3E3604B59B971CA617DF337538D2179EBB22F3BDC9D0C6DA56BA7DDFD205A50
+priv=009D4CBA2BFB1A8330D3E20E59D281D476D231C73A
+ctype=14
+
+[lv2ldr]
+type=SELF
+revision=0000
+version=0004008100000000
+self_type=LV2
+erk=0CAF212B6FA53C0DA7E2C575ADF61DBE68F34A33433B1B891ABF5C4251406A03
+riv=9B79374722AD888EB6A35A2DF25A8B3E
+pub=1034A6F98AF6625CC3E3604B59B971CA617DF337538D2179EBB22F3BDC9D0C6DA56BA7DDFD205A50
+priv=009D4CBA2BFB1A8330D3E20E59D281D476D231C73A
+ctype=14
+
+[lv2ldr]
+type=SELF
+revision=0000
+version=0004008200000000
+self_type=LV2
+erk=0CAF212B6FA53C0DA7E2C575ADF61DBE68F34A33433B1B891ABF5C4251406A03
+riv=9B79374722AD888EB6A35A2DF25A8B3E
+pub=1034A6F98AF6625CC3E3604B59B971CA617DF337538D2179EBB22F3BDC9D0C6DA56BA7DDFD205A50
+priv=009D4CBA2BFB1A8330D3E20E59D281D476D231C73A
+ctype=14
+
+[appldr]
+type=SELF
+revision=0000
+version=0000009200000000
+self_type=APP
+erk=95F50019E7A68E341FA72EFDF4D60ED376E25CF46BB48DFDD1F080259DC93F04
+riv=4A0955D946DB70D691A640BB7FAECC4C
+pub=6F8DF8EBD0A1D1DB08B30DD3A951E3F1F27E34030B42C729C55555232D61B834B8BDFFB07E54B343
+priv=006C3E4CCB2C69A5AD7C6F60448E50C7F9184EEAF4
+ctype=21
+
+[appldr]
+type=SELF
+revision=8000
+version=000000800000000
+self_type=APP
+erk=0000000000000000000000000000000000000000000000000000000000000000
+riv=00000000000000000000000000000000
+pub=00000000000000000000000000000000000000000000000000000000000000000000000000000000
+priv=000000000000000000000000000000000000000000
+ctype=00
+
 [appldr]
 type=SELF
 revision=0001
-version=0003001500000000
+version=0003003000000000
 self_type=APP
 erk=79481839C406A632BDB4AC093D73D99AE1587F24CE7E69192C1CD0010274A8AB
 riv=6F0F25E1C8C4B7AE70DF968B04521DDA
 pub=94D1B7378BAFF5DFED269240A7A364ED68446741622E50BC6079B6E606A2F8E0A4C56E5CFF836526
 priv=003DE80167D2F0E9D30F2145144A558D1174F5410C
-ctype=11
\ No newline at end of file
+ctype=11
```

Comment:
What is the reason for updating the keys? I provided only this key because it was all that was needed to decrypt the '06 binary.

Reply:
Somehow it is not enough. I don't know which keys are exactly used, but leaving only one key results in broken EBOOT. I've already tried multiple keys but they just doesn't work properly. The only working method for now is to include all keys or to ask help to someone who knows how those keys work. :/